app/models/miq_widget_set.rb

```diff
@@ -5,6 +5,10 @@ class MiqWidgetSet < ApplicationRecord
 
   WIDGET_DIR =  File.expand_path(File.join(Rails.root, "product/dashboard/dashboards"))
 
+  def self.default
```

Comment:
I was hoping for a `where` in here, too. But since it is just one default, it works. Thnx for changing from a "scope"

a) Do we need `in_my_region` as well? (not sure)
b) would prefer using a non reserved word (`default`) - maybe `default_widget` ?

otherwise, LGTM - ping me to merge once you change

Reply:
> Do we need in_my_region as well? (not sure)

We don't replicate the `miq_sets` table, so it's not required

> would prefer using a non reserved word (default) - maybe default_widget ?

Maybe `default_widget_set` or `default_dashboard`